config/environments/production.rb

```diff
@@ -45,21 +44,31 @@
   # config.action_cable.url = "wss://example.com/cable"
   # config.action_cable.allowed_request_origins = [ "http://example.com", /http:\/\/example.*/ ]
 
+  # Assume all access to the app is happening through a SSL-terminating reverse proxy.
+  # Can be used together with config.force_ssl for Strict-Transport-Security and secure cookies.
+  # config.assume_ssl = true
+
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = true
```

Comment:
I think we still want to comment this out in case students don't have a plan that allows SSL when they deploy.

IIRC you don't get SSL for free Heroku apps, but maybe Render and Fly.io already allow this, I'm not sure.

Reply:
Good catch. I missed that change.